frontend/src/components/login/RecoverPassword.js

```diff
@@ -0,0 +1,56 @@
+import './RecoverPasswordStyle.css';
+import { useContext } from 'react';
+import slugs from 'resources/slugs';
+import { GlobalContext } from 'hooks/GlobalContext';
+import React from 'react';
+
+function RecoverPassword() {
```

Comment:
Since all other Components that we created are called ...Component, I suggest you change this one to RecoverPasswordComponent in all places as well. 🙂

Reply:
I will change that now..Thanks for reviewing :)

Reply:
you are welcome, thanks for changing it 🙂